packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -71,6 +72,14 @@ class CupertinoButton extends StatefulWidget {
   /// Defaults to null which produces a button with no background or border.
   final Color color;
 
+  /// The color of the button's background when the button is disabled.
+  ///
+  /// Ignored if the [CupertinoButton] doesn't also have a [color].
+  ///
+  /// Defaults to a standard iOS disabled color when [color] is specified and
```

Comment:
Is that defined and static or does it vary by iOS version?

Reply:
Also, disabled color of which elements? And is it for disabled + selected as well?